src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.LowLevelLifoSemaphore.Unix.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Threading
+{
+    internal static partial class ClrThreadPool
+    {
+        /// <summary>
+        /// A LIFO semaphore (used by the thread pool to keep hot threads hot and cold threads cold).
+        /// </summary>
+        private class LowLevelLifoSemaphore
+        {
+            private WaitSubsystem.WaitableObject _semaphore;
+
+            public LowLevelLifoSemaphore(int initialCount, int maxSignalCount)
```

Comment:
nit: use similar naming scheme for initial/max (initialSignalCount and maximumSignalCount are the names used by WaitableObject)